### PR TITLE
Example: Run Secretlint only on changed files with Node.js or Docker

### PR DESCRIPTION
**Pull request by gkzz**
* The followings are example workflows of https://github.com/secretlint/secretlint/issues/166
* The reason I make this pr is I cannot reopen #4 , because I closed it as a draft

### CODE REVIEW

File: .github/workflows/secretlint.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node_version: 16
+          node-version: 16
```
Comment (gkzz) [inline .github/workflows/secretlint.yml:15]: setup-node action.yml says
https://github.com/actions/setup-node/blob/bed538bd04a773f1616efba082915a7638d8eb5f/action.yml#L8
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043844626

File: .github/workflows/secretlint-only-on-changed-files.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - uses: technote-space/get-diff-action@v6
+      - name: setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install
+        run: npm install
+      - name: Lint with Secretlint
+        run: npx --no-install secretlint ${{ env.GIT_DIFF_FILTERED }}
+        if: env.GIT_DIFF && !env.MATCHED_FILES
```
Comment (gkzz) [inline .github/workflows/secretlint-only-on-changed-files.yml:23]: This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043844625

File: .github/workflows/secretlint-via-docker-only-on-changed-files.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        uses: addnab/docker-run-action@v3
+        with:
+          # https://hub.docker.com/r/secretlint/secretlint/tags
+          image: secretlint/secretlint:v5.1.1
+          # https://github.com/secretlint/secretlint#using-docker
+          options: >
+            -v ${{ github.workspace }}:${{ github.workspace }}
+            -w=${{ github.workspace }}
+            --rm
+          run: secretlint ${{ env.GIT_DIFF_FILTERED }} # run only on changed files
```
Comment (gkzz) [inline .github/workflows/secretlint-via-docker-only-on-changed-files.yml:24]: This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043844623

File: aws-secrets [hunk 4]
```diff
@@ -0,0 +1 @@
+AWS_SECRET_ACCESS_KEY = wJalrXUtnFEMI/K7MDENG/bPxRfiCYSECRETSKEY
```
Comment (gkzz) [inline aws-secrets]: Copied from the same file on aws-secret branch
https://github.com/secretlint/secretlint-github-actions-example/blob/1729c7d6f3340734a1936b4d98192c04fba8db31/aws-secrets

**Review by azu — APPROVED** [review]
LGTM.

We need to remove aws-secrets before merging it.
After merge it, it is good that create new seprated example PR like #1

File: .github/workflows/secretlint.yml (hunk 1 above)
Comment (gkzz) [reply .github/workflows/secretlint.yml:15]: I already fixed it, #6

**Comment by gkzz** [comment]
> LGTM.
> 
> We need to remove aws-secrets before merging it. After merge it, it is good that create new seprated example PR like #1

I make another pr as this demo branch like #1

**Pull request merged**